CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [0.7.0](https://github.com/petermetz/cactus/compare/v0.6.0...v0.7.0) (2021-08-04)
+
+
+### Bug Fixes
+
+* **connector-corda:** fix build broken by operationId rename ([291dd3b](https://github.com/petermetz/cactus/commit/291dd3bc666939fffbc3780eaefd9059c756878a))
+* **examples/discounted-cartrade:** update default.json to activate trade process ([dd5c3f1](https://github.com/petermetz/cactus/commit/dd5c3f12e6c08a5798ef5b7f794114242a1b1e94))
+
+
+### Features
+
+* **aws-sm:** added keychain plugin for aws secret manager ([ed6db9e](https://github.com/petermetz/cactus/commit/ed6db9edc2064046308be91b73f620cbb2a6fb58)), closes [#912](https://github.com/petermetz/cactus/issues/912)
+* **azure-kv:** added keychain plugin for azure keyvault ([69e7b50](https://github.com/petermetz/cactus/commit/69e7b50f127fbf247a43288353388782a4301686)), closes [#971](https://github.com/petermetz/cactus/issues/971)
+* **besu:** record locator ([2410d6d](https://github.com/petermetz/cactus/commit/2410d6d4430f5b1f587befa01824a7674c6e11fa))
+* **connector-fabric:** endorsing peers request arg [#1122](https://github.com/petermetz/cactus/issues/1122) ([c6057a7](https://github.com/petermetz/cactus/commit/c6057a7ac508f3bd8da8c9611414a627ff772024)), closes [#1123](https://github.com/petermetz/cactus/issues/1123) [#1130](https://github.com/petermetz/cactus/issues/1130)
+* **connector-fabric:** identity json signing credentials [#1130](https://github.com/petermetz/cactus/issues/1130) ([bc262a2](https://github.com/petermetz/cactus/commit/bc262a24ae654899e8941e7910953c6c561ea778)), closes [#1124](https://github.com/petermetz/cactus/issues/1124)
+* **example:** add README on examples/discounted-cartrade ([2eac8bf](https://github.com/petermetz/cactus/commit/2eac8bf8c819911ded6cbdf2368398fbd6977a64))
+* **examples/discounted_cartrade:** add preferredcustomer-jugdement ([43be168](https://github.com/petermetz/cactus/commit/43be16828581ebf0fabb4b0e8356e052bbcd8760))
+* **google-sm:** added keychain plugin for google secret manager ([1419b2c](https://github.com/petermetz/cactus/commit/1419b2c8c1b12a08e940ba4f0ff824420d88c233)), closes [#983](https://github.com/petermetz/cactus/issues/983)
+
```

Comment:
Would you add the following sentence? The PR related with Issue 1181 includes the codes about Validator Indy.
* **validator-indy:** enhance the request feature on Validator-Indy [#1181](https://github.com/hyperledger/cactus/issues/1181